File: app/src/main/java/com/maye/today/register/RegisterModel.java
package com.maye.today.register;

import com.maye.today.domain.User;

import io.reactivex.rxjava3.core.Observable;
import okhttp3.ResponseBody;

/**
 * RegisterModel接口
 */
public interface RegisterModel {

    Observable<ResponseBody> register(User user);
}

File: LoginModule/src/main/java/com/maye/register/mvp/RegisterModel.kt
package com.maye.register.mvp

import com.maye.base.bean.User
import io.reactivex.rxjava3.core.Observable
import okhttp3.ResponseBody

/**
 * RegisterModel接口
 */
interface RegisterModel {
    fun register(user: User): Observable<ResponseBody>
}